### PR TITLE
Fix/app/footer

### PR DESCRIPTION
**Pull request by MarieTheCoin**

### CODE REVIEW

File: apps/site-app/src/components/Footer/index.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import styles from './styles.module.less';
 
 export const Footer = () => {
   return (
-    <div id={styles.footerContainer} className={styles.desktopContent}>
```
Comment (MarieTheCoin) [inline apps/site-app/src/components/Footer/index.tsx:8]: Didn't find any css for that class anywhere. Removing it fixes the footer problem.

Comment (FrozenKiwi) [reply apps/site-app/src/components/Footer/index.tsx:8]: our churn lets in a lot of bugs. Adding storybook to our testing will help with this.

Comment (MarieTheCoin) [reply apps/site-app/src/components/Footer/index.tsx:8]: Should this be merged?

Comment (FrozenKiwi) [reply apps/site-app/src/components/Footer/index.tsx:8]: yes of course, I generally leave the PR for the author to merge when their happy.  I'll merge this one in now tho!

**Pull request merged**